src/actions/keys.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const alt = require('../alt');
+
+class KeyActions {
+  findNext(evt) {
+    this.dispatch(evt);
+  }
+  findPrevious(evt) {
+    this.dispatch(evt);
+  }
+  moveByScrollUpLine(evt) {
+    this.dispatch(evt);
+  }
+  moveByScrollDownLine(evt) {
+    this.dispatch(evt);
+  }
+  tab(evt) {
```

Comment:
onFindNext, onFindPrevious, onMoveByScollUpLine, onMoveByScrollDownLine, and Indent are all code mirror specific shortcuts - so i think it makes sense to group them all into one action class? called editorCommands maybe? then print would be its own class, called systemCommands?